the-blue-alliance-ios/View Controllers/Settings/SettingsViewController.swift

```diff
@@ -0,0 +1,191 @@
+
+        return cell
+    }
+
+    // MARK: - Table View Delegate
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+
+        switch indexPath.section {
+        case SettingsSection.info.rawValue:
+            switch indexPath.row {
+            case InfoRow.website.rawValue:
+                openTBAWebsite()
+            case InfoRow.repo.rawValue:
+                openGitHub()
+            default:
+                fatalError("This row does not exist")
+            }
+        case SettingsSection.debug.rawValue:
+            switch indexPath.row {
+            case DebugRow.deleteNetworkCache.rawValue:
+                let alertController = UIAlertController(title: "Delete Network Cache", message: "Are you sure you want to delete all the network cache data?", preferredStyle: .alert)
+
+                let deleteCacheAction = UIAlertAction(title: "Delete", style: .destructive) { [weak self] (deleteAction) in
+                    self?.deleteNetworkCache()
+                }
+                let cancelAction = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
+
+                alertController.addAction(deleteCacheAction)
+                alertController.addAction(cancelAction)
+
+                self.present(alertController, animated: true, completion: nil)
+            case DebugRow.deleteAppData.rawValue:
+                let alertController = UIAlertController(title: "Delete App Data", message: "Are you sure you want to delete local app data?", preferredStyle: .alert)
+
+                let deleteDataAction = UIAlertAction(title: "Delete", style: .destructive) { [weak self] (deleteAction) in
+                    self?.deleteAppData()
+                }
+                let cancelAction = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
+
+                alertController.addAction(deleteDataAction)
+                alertController.addAction(cancelAction)
+
+                self.present(alertController, animated: true, completion: nil)
+            default:
+                fatalError("This row does not exist")
+            }
+
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+
+    // MARK: - Private Methods
+
+    internal func openTBAWebsite() {
+        if let url = URL(string: InfoURL.website.rawValue) {
+            openURL(url: url)
+        }
+    }
+
+    internal func openGitHub() {
+        if let url = URL(string: InfoURL.github.rawValue) {
+            openURL(url: url)
+        }
+    }
+
+    private func openURL(url: URL) {
+        print(url)
+        if urlOpener.canOpenURL(url) {
+            urlOpener.open(url, options: [:], completionHandler: nil)
+        }
+    }
+
+    internal func deleteNetworkCache() {
+        TBAKit.clearLastModified()
+    }
+
+    internal func deleteAppData() {
+        persistentContainer.deleteAllEntities()
```

Comment:
If the `NSPersistentContainer` extension gets tests, we can mock this out and just make sure it gets called for this method. Should be easy.